app/contributors/[slug]/page.tsx

```diff
@@ -105,7 +105,7 @@ export default async function Page({ params }: Params) {
           <div className="mt-10 flex flex-col justify-between rounded-lg border border-current bg-amber-300/20 px-4 py-4 font-semibold text-amber-500 dark:bg-amber-500/20 max-sm:mx-3 max-sm:gap-3 sm:flex-row sm:px-6 xl:px-10 ">
             <span className="flex items-center gap-4">
               <FiAlertTriangle size={20} />
-              Contributor profile has not been updated.
+              Bio is missing. Update it to complete your profile!
```

Comment:
"your" is incorrect. We do not know who is looking at the profile. It could be themselves, or someone else.